Networking/Networking/Remote/MediaRemote.swift

```diff
@@ -294,7 +294,7 @@ public extension MediaRemote {
 
     private enum ParameterKey {
         static let pageNumber: String = "page"
-        static let pageSize: String   = "number"
+        static let pageSize: String   = "per_page"
```

Comment:
Thanks for working on this, @hafizrahman! I will wait for @hichamboushaba to review this.

---

I am jumping in as this parameter key change will affect both `loadMediaLibrary` and `loadMediaLibraryFromWordPressSite` methods in this `MediaRemote` file. 

The `number` parameter key is from the following documentation. https://developer.wordpress.com/docs/api/1.2/get/sites/%24site/media/

Please make sure to test the following login cases and update the PR testing instructions accordingly.

1. Jetpack
2. JCP
3. No Jetpack. (Application password)

I am afraid that one of the above cases could be broken/affected by this change. 

---
Nice to have: It would be nice to add a unit test to ensure that the parameter key is correct.

Reply:
Oh good catch, I tested in WordPress.com console and indeed it wants `number`. Let me refactor this.

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/14444/commits/7215568fbe05f767eebc8710a371eceb86007256

Good for another review.